clang/include/clang/CIR/Dialect/IR/CIROps.td

```diff
@@ -702,10 +702,14 @@ def UnaryOp : CIR_Op<"unary", [Pure, SameOperandsAndResultType]> {
   }];
 
   let results = (outs CIR_AnyType:$result);
-  let arguments = (ins Arg<UnaryOpKind, "unary op kind">:$kind, Arg<CIR_AnyType>:$input);
+  let arguments = (ins Arg<UnaryOpKind, "unary op kind">:$kind,
```

Comment:
Should this be documented in the `description` since we're adding a new argument?

Reply:
Yeah. Good catch.